### PR TITLE
refactor(core/webidl): modernize run()

### PR DESCRIPTION
**Pull request by saschanaz**
This:

1. removes deprecated `cb()` and an unused exposed promise `done`.
2. removes jQuery.

### CODE REVIEW

File: src/core/base-runner.js [hunk 1]
```diff
@@ -62,7 +62,7 @@ export async function runAll(plugs) {
     try {
       await task(respecConfig);
     } catch (err) {
-      console.error(err);
+      pub("error", err);
```
Comment (marcoscaceres) [inline src/core/base-runner.js]: I don't think we want to do this, unfortunately: the reason is that we don't want to surface JS errors to end users, as they can't actually fix those. We only `pub("error")` for things users can fix.

**Comment by marcoscaceres** [comment]
The rest looks great tho! thanks for doing that.

File: src/core/base-runner.js (hunk 1 above)
Comment (saschanaz) [reply src/core/base-runner.js]: I still think we should have a way to show that something failed in a fatal way. A silent failure is never helpful for both authors and devs.

Comment (saschanaz) [reply src/core/base-runner.js]: Maybe something like `pub("fatal", err)`?

Comment (saschanaz) [reply src/core/base-runner.js]: I'll revert that commit in this PR, we can discuss more in #1760.

Comment (sidvishnoi) [reply src/core/base-runner.js]: Related: https://github.com/w3c/respec/issues/1741

**Pull request merged**